server/config/config.go

```diff
@@ -128,9 +128,7 @@ func New(flags *pflag.FlagSet, logger logger) (*Config, error) {
 
 	configOptions.registerDefaults(vp)
 
-	warnAboutDeprecatedFields(vp, logger)
```

Comment:
Do we want to remove these deprecation warnings?

Reply:
Or is it related to this PR: https://github.com/kubeshop/tracetest/pull/2082?

Reply:
no, it's a mistake

Reply:
@danielbdias i was trying to fix that on this pr, but decided to make another pr for that, and mistakenly removed this line. It's readded noe